.vscode/settings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This should only be local, right? 👀

Reply:
It should. I added `.vscode` to `.gitignore` in https://github.com/getnacelle/nacelle-react/pull/8/commits/ce36d87006615909ec43287674f7c68f2c18d2ba, but I need to delete it too. Will fix!